<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloaderDialog_V1</class>
 <widget class="QDialog" name="DownloaderDialog_V1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Download Example</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_DownloadPath">
     <property name="text">
      <string>Download To:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_DownloadURL">
     <property name="text">
      <string>Download Url</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_OpenSaveFile">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_DownloadPath">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_DownloadURL">
     <property name="inputMethodHints">
      <set>Qt::ImhUrlCharactersOnly</set>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_Satus">
     <property name="text">
      <string>User Guide:
1.Input download URL.
2.Click Browse button to give file name.
3.Click Save button.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
